index.src.html

```diff
@@ -619,7 +619,63 @@ <h3 id="data-minimization">
     Data Minimization
   </h3>
 
-  TODO
+  Minimization is a strategy that involves exposing as little information to
+  other communication partners as is required for a given operation to
+  complete. More specifically, it requires not providing access to more
+  information than was apparent in the user-mediated access or allowing the
+  user some control over which information exactly is provided.
+
+  For example, if the user has provided access to a given file, the object
+  representing that should not make it possible to obtain information about
+  that file's parent directory and its contents as that is clearly not what is
+  expected.
+
+  Basic fingerprinting guidance can be found here.
+
+  In context of data minimization it is natural to ask what data is passed
+  around between the different parties, how persistent the data items and
+  identifiers are, and whether there are correlation possibilities between
+  different protocol runs.
+
+  For example, the W3C Device APIs Working Group has defined the following
+  requirements in their Device API Privacy Requirements document.
+
+  *   APIs must make it easy to request as little information as required for
+      the intended usage. For instance, an API call should require specific
```

Comment:
We can simply link to this document, or provide the long quotes in an annex or so (we already got some feedback about the document being lengthy)

Reply:
Should be addressed by https://github.com/w3cping/security-questionnaire/commit/062d59217e25994124100d045e4f0eff64cab68f now.